.flowconfig

```diff
@@ -0,0 +1,9 @@
+[ignore]
```

Comment:
Having a config file like this live in the project seems to diverge from keeping config files in mastarm.  But it does sorta seem like this file must exist in order for flow to run, so maybe it's ok.